tests/examples/aot_mlp_test.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Nod Labs, Inc
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import logging
+from pathlib import Path
+import sys
+import subprocess
+import unittest
+
+REPO_DIR = Path(__file__).resolve().parent.parent.parent
```

Comment:
I kind of hate this but don't know a better way

Reply:
Yeah.